lightning/src/blinded_path/payment.rs

```diff
@@ -272,13 +272,6 @@ pub(crate) enum BlindedPaymentTlvs {
 	Receive(ReceiveTlvs),
 }
 
-// Used to include forward and receive TLVs in the same iterator for encoding.
-#[derive(Clone)]
-enum BlindedPaymentTlvsRef<'a> {
-	Forward(&'a ForwardTlvs),
-	Receive(&'a ReceiveTlvs),
-}
-
```

Comment:
Let's instead keep this for max length calculation and add a `BlindedPaymentTlvsMut` for padding, as discussed offline.